This field in </mnt/sdc1/sbini/command_interaction/training/settings/SCR_conf.py.py> file rise an ERROR because the fold <lightning_logs/SCR/eng/GaussCL_PEM_v1/checkpoints> doesn't exist
Loaded </mnt/sdc1/sbini/command_interaction/training/settings/SCR_conf.py> as configuration file.
CUDA acceleration available on <1> devices: <[0]>
Global seed set to 2
The TensorBoard logggers will be saved in <lightning_logs/SCR/eng/conformer/GaussCL_PEM_v1>.
GPU available: True (cuda), used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
The following callbacks returned in `LightningModule.configure_callbacks` will override existing callbacks passed to Trainer: ModelCheckpoint
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name    | Type             | Params
---------------------------------------------
0 | model   | Conformer        | 136 K 
1 | output  | Linear           | 1.3 K 
2 | softmax | Softmax          | 0     
3 | loss_fn | CrossEntropyLoss | 0     
---------------------------------------------
137 K     Trainable params
0         Non-trainable params
137 K     Total params
0.550     Total estimated model params size (MB)
Training: 0it [00:00, ?it/s]Training:   0%|          | 0/53 [00:00<?, ?it/s]Epoch 0:   0%|          | 0/53 [00:00<?, ?it/s] 